.ci/build.sh

```diff
@@ -1,11 +1,53 @@
 #!/bin/bash
 set -e
 
+# set to one of the following values
+# - don't use HelmQA: (empty)
+# - use HelmQA in SaaS mode: 'http://helmqa-zhaw-prod-demos.appuioapp.ch/livecheck?repo'
+# - use HelmQA locally in Docker mode: 'http://127.0.0.1:5000/livecheck?repo'
+HELMQA_URL='http://helmqa-zhaw-prod-demos.appuioapp.ch/livecheck?repo'
 REPO_URL="${REPO_URL:-https://charts.appuio.ch}"
 echo "----> Deploying to $REPO_URL"
 
 GIT_REPO="$(git config remote.origin.url)"
 
+if [ -n "$HELMQA_URL" ]; then
+  did=
+  if [[ "$HELMQA_URL" =~ 127.0.0.1 ]]; then
+    docker pull jszhaw/helmqa
+    did="$(docker run -d -p 127.0.0.1:5000:5000 helmqa)"
```

Comment:
Must be `jszhaw/helmqa` instead of `helmqa`.

I would also recommend to use `docker.io/jszhaw/helmqa` here and in the pull command above. This prevents issues with systems that have multiple docker repositories configured.

Reply:
done